<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="8" failed="0" ignored="0" total="8" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="AmazonShopSuite" duration-ms="17739" started-at="2019-03-13T12:34:16Z" finished-at="2019-03-13T12:34:33Z">
    <groups>
    </groups>
    <test name="SmokeTest" duration-ms="17739" started-at="2019-03-13T12:34:16Z" finished-at="2019-03-13T12:34:33Z">
      <class name="tests.SmokeTestCase">
        <test-method status="FAIL" signature="SetupAmazon()[pri:0, instance:tests.SmokeTestCase@4c70fda8]" name="SetupAmazon" is-config="true" duration-ms="17687" started-at="2019-03-13T14:34:16Z" finished-at="2019-03-13T14:34:33Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Account List element was not found]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Account List element was not found
	at org.testng.Assert.fail(Assert.java:93)
	at amazon.Pages.LoginPages.Login(LoginPages.java:37)
	at amazon.AmazonBase.SetupAmazon(AmazonBase.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:166)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetupAmazon -->
        <test-method status="SKIP" signature="AddtoCartSmoke()[pri:0, instance:tests.SmokeTestCase@4c70fda8]" name="AddtoCartSmoke" duration-ms="1" started-at="2019-03-13T14:34:33Z" description="Test adds an item to the cart, SMOKE" finished-at="2019-03-13T14:34:33Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Account List element was not found]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Account List element was not found
	at org.testng.Assert.fail(Assert.java:93)
	at amazon.Pages.LoginPages.Login(LoginPages.java:37)
	at amazon.AmazonBase.SetupAmazon(AmazonBase.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:166)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AddtoCartSmoke -->
        <test-method status="SKIP" signature="SubmitAddressSmoke()[pri:0, instance:tests.SmokeTestCase@4c70fda8]" name="SubmitAddressSmoke" duration-ms="0" started-at="2019-03-13T14:34:33Z" description="Test fills address form and submits it, SMOKE" depends-on-methods="tests.SmokeTestCase.AddtoCartSmoke" finished-at="2019-03-13T14:34:33Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method SmokeTestCase.SubmitAddressSmoke()[pri:0, instance:tests.SmokeTestCase@4c70fda8] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method SmokeTestCase.SubmitAddressSmoke()[pri:0, instance:tests.SmokeTestCase@4c70fda8] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1084)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SubmitAddressSmoke -->
        <test-method status="SKIP" signature="FillCCSmoke()[pri:0, instance:tests.SmokeTestCase@4c70fda8]" name="FillCCSmoke" duration-ms="0" started-at="2019-03-13T14:34:33Z" description="Test proceeds to the payment and fills CC information, SMOKE" depends-on-methods="tests.SmokeTestCase.AddtoCartSmoke, tests.SmokeTestCase.SubmitAddressSmoke" finished-at="2019-03-13T14:34:33Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method SmokeTestCase.FillCCSmoke()[pri:0, instance:tests.SmokeTestCase@4c70fda8] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method SmokeTestCase.FillCCSmoke()[pri:0, instance:tests.SmokeTestCase@4c70fda8] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1084)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- FillCCSmoke -->
        <test-method status="SKIP" signature="CloseBrowser()[pri:0, instance:tests.SmokeTestCase@4c70fda8]" name="CloseBrowser" is-config="true" duration-ms="0" started-at="2019-03-13T14:34:33Z" finished-at="2019-03-13T14:34:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CloseBrowser -->
      </class> <!-- tests.SmokeTestCase -->
      <class name="tests.AddMusicTestCase">
        <test-method status="SKIP" signature="SetupAmazon()[pri:0, instance:tests.AddMusicTestCase@46d56d67]" name="SetupAmazon" is-config="true" duration-ms="0" started-at="2019-03-13T14:34:33Z" finished-at="2019-03-13T14:34:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetupAmazon -->
        <test-method status="SKIP" signature="FindItem()[pri:0, instance:tests.AddMusicTestCase@46d56d67]" name="FindItem" is-config="true" duration-ms="0" started-at="2019-03-13T14:34:33Z" finished-at="2019-03-13T14:34:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- FindItem -->
        <test-method status="SKIP" signature="Add5VinylssToCart()[pri:0, instance:tests.AddMusicTestCase@46d56d67]" name="Add5VinylssToCart" duration-ms="0" started-at="2019-03-13T14:34:33Z" description="Test added and removed 5 Audio CDs" finished-at="2019-03-13T14:34:33Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Account List element was not found]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Account List element was not found
	at org.testng.Assert.fail(Assert.java:93)
	at amazon.Pages.LoginPages.Login(LoginPages.java:37)
	at amazon.AmazonBase.SetupAmazon(AmazonBase.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:166)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Add5VinylssToCart -->
        <test-method status="SKIP" signature="AddCDToCart()[pri:0, instance:tests.AddMusicTestCase@46d56d67]" name="AddCDToCart" duration-ms="0" started-at="2019-03-13T14:34:33Z" description="Add all types of music items and removed them from the carts" finished-at="2019-03-13T14:34:33Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Account List element was not found]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Account List element was not found
	at org.testng.Assert.fail(Assert.java:93)
	at amazon.Pages.LoginPages.Login(LoginPages.java:37)
	at amazon.AmazonBase.SetupAmazon(AmazonBase.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:166)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AddCDToCart -->
        <test-method status="SKIP" signature="AddCassetteToCart()[pri:0, instance:tests.AddMusicTestCase@46d56d67]" name="AddCassetteToCart" duration-ms="0" started-at="2019-03-13T14:34:33Z" description="Add all types of music items and removed them from the carts" finished-at="2019-03-13T14:34:33Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Account List element was not found]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Account List element was not found
	at org.testng.Assert.fail(Assert.java:93)
	at amazon.Pages.LoginPages.Login(LoginPages.java:37)
	at amazon.AmazonBase.SetupAmazon(AmazonBase.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:166)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AddCassetteToCart -->
        <test-method status="SKIP" signature="AddMp3ToCart()[pri:0, instance:tests.AddMusicTestCase@46d56d67]" name="AddMp3ToCart" duration-ms="0" started-at="2019-03-13T14:34:33Z" description="Add all types of music items and removed them from the carts" finished-at="2019-03-13T14:34:33Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Account List element was not found]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Account List element was not found
	at org.testng.Assert.fail(Assert.java:93)
	at amazon.Pages.LoginPages.Login(LoginPages.java:37)
	at amazon.AmazonBase.SetupAmazon(AmazonBase.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:166)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AddMp3ToCart -->
        <test-method status="SKIP" signature="AddVinylToCart()[pri:0, instance:tests.AddMusicTestCase@46d56d67]" name="AddVinylToCart" duration-ms="0" started-at="2019-03-13T14:34:33Z" description="Add all types of music items and removed them from the carts" finished-at="2019-03-13T14:34:33Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Account List element was not found]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Account List element was not found
	at org.testng.Assert.fail(Assert.java:93)
	at amazon.Pages.LoginPages.Login(LoginPages.java:37)
	at amazon.AmazonBase.SetupAmazon(AmazonBase.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:166)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AddVinylToCart -->
        <test-method status="SKIP" signature="CloseBrowser()[pri:0, instance:tests.AddMusicTestCase@46d56d67]" name="CloseBrowser" is-config="true" duration-ms="0" started-at="2019-03-13T14:34:33Z" finished-at="2019-03-13T14:34:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CloseBrowser -->
      </class> <!-- tests.AddMusicTestCase -->
    </test> <!-- SmokeTest -->
  </suite> <!-- AmazonShopSuite -->
</testng-results>
